native-filesystem/source/native_filesystem.cc

```diff
@@ -112,6 +118,7 @@ void NativeFS::flushBlock(int block_index) {
 }
 
 void NativeFS::freeRange(uint64_t start, uint64_t end) {
+  LOG(INFO) << "[freeRange] start: " << start << " end: " << end;
```

Comment:
Is this log line critical? It seems to be generating a lot of stdout noise in the build

Reply:
No. I will remove it. I was debugging this line.